### ISSUE TITLE
 Monitor cross-validation curve

### ISSUE THREAD

**Issue by ahmad-hl** [godka/Pensieve-PPO#8]
I was trying to monitor the `TD_loss and rewards `using **tensorboard** by launching it using this command:
`sim> tensorboard logdir=./results`
However, I couldn't see the **validation curve** of the `rl-test.py`
I can only see the  `TD_loss, entropy and rewards` of `train.py`.
How can I enable and show the cross-validation curve to monitor the model convergence?

**Comment by godka** [comment]
hi, the tensorboard has already plotted the performance curve on the validation set.

Your can customize your information in

https://github.com/godka/Pensieve-PPO/blob/ad4d194867cf63b2c69cb748d81a7b52e6d03ec8/src/train.py#L149

**Comment by ahmad-hl** [comment]
Thank you 

Do you have published papers with these two contributions?
I mean Pensieve/PPO and Pensieve/SAC! Please share papers if so.
How was the performance of Pensieve/SAC compared to Pensieve/PPO?

**Comment by godka** [comment]
Nop, we have not published papers yet. They are just open-sourced projects.

Regarding to the performance, we have tried our best to check codes and find proper hyper-parameters for Pensieve-SAC. Unfortunately, it didn't perform well as we expected. The overall performance is about 16% lower than Pensieve-PPO (0.82 vs. 0.98). We doubt the maximum entropy algorithm (e.g., SAC) is not sufficient for the ABR task.

Feel free to let us know if you have other opinions :).

**Comment by ahmad-hl** [comment]
Thank you for sharing such valuable information 
I will share my finding once I have a full understanding of these RL concepts and do experiments

Why do you have `ABRenv.py` & `abr.py`?
It feels they are doing a similar job which is interacting with the environment and retrieving the new state!

**Comment by ahmad-hl** [comment]
I forked your project to support `Python 3.8 & TensorFlow 2.7`, and integrate and fix `sac `errors.
You can have a look at the following repo:

> https://github.com/ahmad-hl/Pensieve-PPO

I'm looking forward to your answer to the above question.
